iscsi/targetd/provisioner/iscsi-provisioner.go

```diff
@@ -19,7 +19,8 @@ package provisioner
 import (
 	"errors"
 	"fmt"
-	"github.com/Sirupsen/logrus"
+	"github.com/golang/glog"
+	//"github.com/Sirupsen/logrus"
```

Comment:
Do you want to keep this `logrus` line by commenting out? I'd prefer to clean it up.

Reply:
this should be fixed now